### PR TITLE
header and footer

### PR DESCRIPTION
**Pull request by 2mnyzs**
Added config, header, footer, and normalize.css.

### CODE REVIEW

File: server/footer.php [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<footer>
+	<script src='/assets/js/jquery-3.1.1.min.js'></script>
+	<script src='/assets/js/queue.js'></script>
```
Comment (MatthewMeacham) [inline server/footer.php]: Remove this

File: server/header.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<!doctype html>
```
Comment (MatthewMeacham) [inline server/header.php]: Capitalize DOCTYPE >:(

File: server/header.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+<html class="no-js" lang="">
```
Comment (MatthewMeacham) [inline server/header.php]: should lang really be empty? lang="en"?

File: server/footer.php [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<footer>
+	<script src='/assets/js/jquery-3.1.1.min.js'></script>
+
```
Comment (JakeKo) [inline server/footer.php]: Whitespace?

**Pull request merged**